src/main.cpp

```diff
@@ -53,6 +54,16 @@ llvm::cl::opt<bool> arg_associative_sum("associative",
                  "(experimental)"),
   llvm::cl::init(false));
 
+llvm::cl::opt<bool> arg_add_identity("add-identity",
+  llvm::cl::desc("Assume that fpadd 0.0 is an identity operation "
+                 "(experimental)"),
+  llvm::cl::init(false));
+
+llvm::cl::opt<bool> arg_mul_identity("mul-identity",
+  llvm::cl::desc("Assume that fpmul 1.0 is an identity operation "
+                 "(experimental)"),
+  llvm::cl::init(false));
+
```

Comment:
Seongwon suggested that since these are 'safe' operations, we can use this 'rules' as an default behavior, not as options.
I think it makes sense, too. But for now I made them optional just in case.